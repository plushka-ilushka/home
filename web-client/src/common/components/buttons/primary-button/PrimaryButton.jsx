import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';

import './primary-button.scss';
import Button from '../main-button/Button';

export default function PrimaryButton(props) {
  const { className, ...restProps } = props;

  return (
    <Button
      className={classNames('primary-button', className)}
      {...restProps}
    />
  );
}

PrimaryButton.propTypes = {
  className: PropTypes.string,
};

PrimaryButton.defaultProps = {
  className: undefined,
};
