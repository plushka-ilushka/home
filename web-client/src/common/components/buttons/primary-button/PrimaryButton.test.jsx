import React from 'react';
import { mount } from 'enzyme';
import PrimaryButton from './PrimaryButton';

describe('PaintedButton Component', () => {
  it('PaintedButton Snapshot', () => {
    expect(mount(<PrimaryButton />)).toMatchSnapshot();
  });
});
