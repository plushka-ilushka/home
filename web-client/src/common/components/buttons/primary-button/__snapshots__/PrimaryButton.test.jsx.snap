// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaintedButton Component PaintedButton Snapshot 1`] = `
<PrimaryButton>
  <Button
    className="primary-button"
    type="button"
  >
    <button
      className="button primary-button"
      type="button"
    />
  </Button>
</PrimaryButton>
`;
